tests/grain_size_yield.prm

```diff
@@ -19,6 +19,7 @@ subsection Material model
     set Reference temperature                = 293
     set Grain growth rate constant           = 0
     set Work fraction for boundary area change = 0
+    set Work fraction for boundary area change when yielding = 0
```

Comment:
So when setting the new parameter to 0 nothing changes, which is good. But we also need a test that uses the new feature.